pipelines/sbom-tool-main-build.yaml

```diff
@@ -0,0 +1,360 @@
+  parameters:
+    sdl:
+      sourceAnalysisPool:
+        name: sbom-windows-build-pool
+        os: windows
+    stages:
+      - stage: stage1
+        jobs:
+        - job: Job_1
+          displayName: Build (Windows + NuGet)
+          templateContext:
+            outputs:
+            - output: pipelineArtifact
+              targetPath: $(Build.ArtifactStagingDirectory)
+              artifactName: '$(OutputArtifactName)'
+          pool:
+            name: sbom-windows-build-pool
+            os: windows
+          steps:
+          - task: UseDotNet@2
+            displayName: 'Use .NET Core'
+            inputs:
+              useGlobalJson: true
+
+          - task: DotNetCoreCLI@2
+            displayName: 'Restore solution'
+            inputs:
+              command: restore
+              feedsToUse: config
+              nugetConfigPath: nuget.config
+              verbosityRestore: Normal
+
+          - task: DotNetCoreCLI@2
+            displayName: Build
+            inputs:
+              arguments: '-c $(BuildConfiguration)'
+
+          - task: UseDotNet@2
+            displayName: 'Install .NET 6 SDK'
+            inputs:
+              version: 6.0.400
+
+          - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@3
+            displayName: 'Sign dlls'
+            inputs:
+              ConnectedServiceName: 'Sbom tool CodeSigning ESRP  service connection'
+              FolderPath: '$(Build.SourcesDirectory)/src'
+              Pattern: |
+                **/Release/**/*.exe
+                **/Release/**/*.dll
+                UseMinimatch: true
+                signConfigType: inlineSignParams
+                inlineOperation: |
+                [
+                        {
+                            "KeyCode" : "CP-230012",
+                            "OperationCode" : "SigntoolSign",
+                            "Parameters" : {
+                                "OpusName" : "Microsoft",
+                                "OpusInfo" : "http://www.microsoft.com",
+                                "FileDigest" : "/fd \"SHA256\"",
+                                "PageHash" : "/NPH",
+                                "TimeStamp" : "/tr \"http://rfc3161.gtm.corp.microsoft.com/TSS/HttpTspServer\" /td sha256"
+                            },
+                            "ToolName" : "sign",
+                            "ToolVersion" : "1.0"
+                        },
+                        {
+                            "KeyCode" : "CP-230012",
+                            "OperationCode" : "SigntoolVerify",
+                            "Parameters" : {},
+                            "ToolName" : "sign",
+                            "ToolVersion" : "1.0"
+                        }
+                ]
+            condition: and(succeeded(), startswith(variables['Build.SourceBranch'], 'refs/tags/'))
+
+          - task: DotNetCoreCLI@2
+            displayName: 'Pack NuGet package'
```

Comment:
This task is disabled (both in the original pipeline and here), do we need it in the yaml?